server/main.lua

```diff
@@ -143,10 +143,11 @@ end)
 local function sendDoctorAlert()
 	if doctorCalled then return end
 	doctorCalled = true
-	local _, doctors = QBCore.Functions.GetDutyCountJob('ambulance')
-	for i = 1, #doctors do
-		local doctor = doctors[i]
-		TriggerClientEvent('ox_lib:notify', doctor.PlayerData.source, { description = Lang:t('info.dr_needed'), type = 'inform' })
```

Comment:
this doesn't work?

Reply:
It didn't and I did not check the function, I see that it should return players as the second value which we are getting in pairs here, I'll check further, might just be the way that playerdata.source is pulling the number, but I was getting that doctor. was a number not a table.

Reply:
corrected